docs/api/ConnectionOpenInPartition.md

```diff
@@ -0,0 +1,65 @@
+ConnectionOpenInPartition function
+======
+
+Creates a new connection in a specific partition.
+
+# Syntax
+
+```C
+typedef
+_IRQL_requires_max_(DISPATCH_LEVEL)
+QUIC_STATUS
+(QUIC_API * QUIC_CONNECTION_OPEN_IN_PARTITION_FN)(
+    _In_ _Pre_defensive_ HQUIC Registration,
+    _In_ uint16_t PartitionIndex,
+    _In_ _Pre_defensive_ QUIC_CONNECTION_CALLBACK_HANDLER Handler,
+    _In_opt_ void* Context,
+    _Outptr_ _At_(*Connection, __drv_allocatesMem(Mem)) _Pre_defensive_
+        HQUIC* Connection
+    );
+```
+
+# Parameters
+
+`Registration`
+
+The valid handle to an open registration object.
+
+`PartitionIndex`
+
+An index into the global partition set.
+
+`Handler`
+
+A pointer to the app's callback handler to be invoked for all connection events.
+
+`Context`
+
+The app context pointer (possibly null) to be associated with the connection object.
+
+`Connection`
+
+On success, returns a handle to the newly opened connection object.
+
+# Return Value
+
+The function returns a [QUIC_STATUS](QUIC_STATUS.md). The app may use `QUIC_FAILED` or `QUIC_SUCCEEDED` to determine if the function failed or succeeded.
+
+# Remarks
+
+`ConnectionOpenInPartition` is used to create a connection in the client application, with an explicit partition index. In server applications, [ListenerOpen](ListenerOpen.md) and [ListenerStart](ListenerStart.md) must be called to listen for incoming connection attempts, and the server side Connection is created in the `QUIC_LISTENER_EVENT_NEW_CONNECTION` event.
+
+'ConnectionOpen' only allocates the resources for the connection, it does not start the connection. To start the connect, the application must call [ConnectionStart](ConnectionStart.md).
```

Comment:
Should `ConnectionOpen` be replaced throughout this document?
Otherwise, we could redirect to the `ConnectionOpen` documentation and only mention the differences here.

Reply:
Updated